backend/models/reviews.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+
+const comment = new mongoose.Schema({
+  lastReviewed: {
+    type: String
+  },
+  value: {
+    type: String
+  }
+});
+
+const section = new mongoose.Schema({
+  id: {
+    type: String
+  },
+  notes: [comment],
+  rating: {
+    type: Number
+  }
+});
+
+const reviewSchema = new mongoose.Schema(
+  {
+    submissionId: {
+      type: mongoose.Schema.ObjectId
+    },
+    programId: {
+      type: mongoose.Schema.ObjectId
+    },
+    userId: {
+      type: String
+    },
+    rating: {
+      type: Number
+    },
+    comments: [comment],
+    lastReviewed: {
+      type: String
+    },
+    questionList: [section]
```

Comment:
should this be named `sections` or `sectionList` instead?

Reply:
I am leaving it as questionList as that is what we used for legacy programs

And I reused frontend components called this so I didn't bother adding a conditional for legacy vs not